src/ASM/ASMs3D.C

```diff
@@ -1535,53 +1535,45 @@ bool ASMs3D::updateCoords (const Vector& displ)
 }
 
 
-void ASMs3D::getBoundaryNodes (int lIndex, IntVec& nodes) const
+void ASMs3D::getBoundaryNodes (int lIndex, IntVec& nodes, int basis) const
 {
+  if (basis == 0)
+    basis = 1;
+
   if (!svol) return; // silently ignore empty patches
 
```

Comment:
for tilsvarende 2D-metode brukes getBasis(basis), mens her går du på svol direkte. Ulogisk, eller forglemmelse? Ellers gjelder de andre kommentarene for 2D-metoden her også.